src/keys/utils.ts

```diff
@@ -41,18 +47,71 @@ async function generateKeyPair(bits: number): Promise<KeyPair> {
   return await generateKeyPair({ bits });
 }
 
+function generateMnemonic(words: 12 | 24 = 24): string {
+  switch (words) {
+    case 12:
+      return Bip39.generateMnemonic(128);
+    case 24:
+    default:
```

Comment:
This should not be necessary here. Just replace the the switch case if an if else. TS will take care of it.

Reply:
This is now using `never` from `src/utils.ts`. This is a sort of "absurd" evaluation. Use this to tell TS that it cannot happen. It's known as an "exhaustive checking".